packages/react/src/utils/errorInDevMode.ts

```diff
@@ -0,0 +1,7 @@
+import { isDevelopmentEnvironment } from '@clerk/shared';
+
+export const errorInDevMode = (message: string) => {
```

Comment:
🙃 When i read the name i thought this would return a boolean
```suggestion
export const logErrorInDevMode = (message: string) => {
```